package.json

```
@@ -20,6 +20,7 @@
   "homepage": "https://github.com/asyncapi/nodejs-template#readme",
   "scripts": {
     "test": "rimraf test/temp && jest --modulePathIgnorePatterns='./template'",
+    "test.updateSnapshot": "rimraf test/temp && jest --updateSnapshot --modulePathIgnorePatterns='./template'",
```

```suggestion
    "test:updateSnapshot": "rimraf test/temp && jest --updateSnapshot --modulePathIgnorePatterns='./template'",
```

just to be consistent with other scripts